journal/api/src/db/schema.ts

```diff
@@ -8,5 +8,14 @@ export const people = sqliteTable('people', {
   avatar: text('avatar')
 })
 
+export const locations = sqliteTable('locations', {
+  id: integer('id').primaryKey({autoIncrement: true}),
+  name: text('name').notNull(),
+  notes: text('notes'),
+  avatar: text('image')
```

Comment:
Should we make the coords their own field(s)?

Reply:
Copilot suggested it, then I was like why would we? but I renamed it to Notes and left the suggested text in there. I don't know if we would rather find a map and use x,y coords instead. Let's chat more.